lib/delayed_resque/performable_method.rb

```diff
@@ -43,12 +50,24 @@ def self.perform(options)
       loaded_object.send(method, *arg_objects)
     end
 
-    def store
-      hsh = {"obj" => @object, "method" => @method, "args" => @args}.merge(@options[:params] || {})
-      unless @options[:unique] || @options[:throttle] || @options[:at] || @options[:in]
-        hsh["t"] = Time.now.to_f
+    def self.before_perform_remove_tracked_jobs(args)
```

Comment:
Was this an existing bug? Not sure what would make this change necessary.

Reply:
This was moved here from `PerformableAsUser` in aha-app because we add the tracking in this gem (in message_sending.rb) so it makes more sense to untrack here too.